### ISSUE TITLE
Search does not work with latest Docusaurus v2

### ISSUE THREAD

**Issue by timothyerwin** [praveenn77/docusaurus-lunr-search#85]
I followed the instructions on the README but the lunr apis return no results...

**Comment by timothyerwin** [comment]
The issue is in https://github.com/praveenn77/docusaurus-lunr-search/blob/master/src/html-to-doc.js#L24

It seems that it can't read the new Docusaurus v2 build structure

**Comment by haskjold** [comment]
I have the same issue. If I reverted the [change](https://github.com/praveenn77/docusaurus-lunr-search/pull/74/files#diff-3274f1a37032fb0ae4e2823def0007c634e869ae0dfc304ff6a12c36513c3a52) done to `utils.js` in https://github.com/praveenn77/docusaurus-lunr-search/pull/74 it works again.

With the change done in PR #74 it generates a list of file paths that does not match what Docusaurus 2.0 generates.

With Docusaurus 2.0.1 I get paths like this:

`build/docs/foo/bar/index.html`

but docusaurus-lunr-search now creates a list of paths for lunr to index that looks like this:

`build/docs/foo/bar.html`

I believe that part of PR #74 should to be reverted.

**Comment by timothyerwin** [comment]
yes, but to be more technically correct, we should not revert #74 entirely but a new pr should be submitted to revert the change to `utils.js`

**Comment by timothyerwin** [comment]
I guess you already said that lol...reread your post in more detail and still drinking coffee ☕ lol

@haskjold I'm just going to fork this repo and revert until it gets unblocked with a new release.

**Comment by noraj** [comment]
In docusaurus v2.1.0 the search bar doesn't display even when having this navbar item:

```js
        {
          type: 'search',
          position: 'right',
        }
```

**Comment by Stavitsky** [comment]
I have the same problem w/ search described by @haskjold.

Versions:
* "@docusaurus/core": "2.1.0",
*  "docusaurus-lunr-search": "2.2.0"

**Comment by paksu** [comment]
Same problem here. Search is broken for versions of docusaurus-lunr-search > 2.1.5

**Comment by fregayeg** [comment]
Hello! I have same problem with Docusaurus v2. Please help 😥

**Comment by praveenn77** [comment]
new version 2.3.0 is published
